consensus/istanbul/validator/weighted.go

```diff
@@ -578,7 +578,7 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, chainI
 		return err
 	}
 
-	newStakingInfo := reward.GetStakingInfoFromStakingCache(blockNum + 1)
+	newStakingInfo := stakingManager.GetStakingInfo(blockNum + 1)
 
 	valSet.stakingInfo = newStakingInfo
 	if valSet.stakingInfo == nil {
```

Comment:
`valSet.stakingInfo == nil` means `newStakingInfo` is nil. Is it okay to access `newStakingInfo.UseGini` in the following code?

Reply:
it returns err if valSet.stakingInfo == nil :)